### PR TITLE
Refactor module with Runtime API changes

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: log-ballerina/src/integration_tests/tests/tests.bal [hunk 1]
```diff
@@ -33,6 +33,9 @@ public function testBasicLogFunctionality() {
     io:ReadableCharacterChannel sc = new (readableResult, UTF_8);
     string outText = checkpanic sc.read(100000);
     string[] logLines = stringutils:split(outText, "\n");
+        foreach var logLine in logLines {
```
Comment (MadhukaHarith92) [inline log-ballerina/src/integration_tests/tests/tests.bal:36]: This is temporary.

**Comment by MadhukaHarith92** [comment]
Need to re-enable test cases by fixing https://github.com/ballerina-platform/ballerina-standard-library/issues/592

**Comment by niveathika** [comment]
@MadhukaHarith92 It seems this is fixed, shall we enable the test cases and merge?

**Comment by MadhukaHarith92** [comment]
> @MadhukaHarith92 It seems this is fixed, shall we enable the test cases and merge?

Yes, it is fixed now. Re-enabled the tests.

**Review by niveathika — APPROVED** [review]

**Pull request merged**